doc/api/child_process.md

```diff
@@ -849,6 +849,10 @@ added: v0.5.9
 The `'message'` event is triggered when a child process uses [`process.send()`][]
 to send messages.
 
+*Note*: since the message goes through JSON serialization and parsing, it might
+not be the same as what is originally sent, see notes of
+[the `JSON.stringify()` specification][`JSON.stringify` spec].
```

Comment:
The MDN documentation doesn't mention the `Nan`/`Infinity` -> `null` thing which I think is one of the surprising behaviors..the link is not about the serialization process, but rather, the caveats, which is described in the notes of the ECMA spec.